.github/CODEOWNERS

```diff
@@ -51,6 +53,7 @@
 /enterpriseknowledgegraph/**/*         @GoogleCloudPlatform/dee-data-ai @GoogleCloudPlatform/python-samples-reviewers
 /eventarc/**/*                         @GoogleCloudPlatform/aap-dpes  @GoogleCloudPlatform/python-samples-reviewers
 /error_reporting/**/*                  @GoogleCloudPlatform/python-samples-reviewers
+/eventarc/**/*                         @GoogleCloudPlatform/aap-dpes  @GoogleCloudPlatform/python-samples-reviewers
```

Comment:
aap-dpes -> torus-dpe

Reply:
(I get you're just moving the line but if we are moving, might as well update to the current name)

Reply:
I have updated all instances of aap-dpes to torus-dpe taking into account the members of both teams are the same. I hope this is what you meant. If you wanted me to change only for this single line PLMK and will update accordingly.